### PR TITLE
Fix Bug in Issue Config when only contact links are set

### PR DESCRIPTION
**Pull request by JakobDev**
Blank Issues should be enabled if they are not explicit disabled through the `blank_issues_enabled` field of the Issue Config. The Implementation has currently a Bug: If you create a Issue Config file with only `contact_links` and without a  `blank_issues_enabled` field,  `blank_issues_enabled` is set to false by default.

The fix is only one line, but I decided to also improve the tests to make sure there are no other problems with the Implementation.

This is a bugfix, so it should be backported to 1.20.

### CODE REVIEW

File: services/issue/template.go [hunk 1]
```diff
@@ -72,7 +72,7 @@ func GetTemplateConfig(gitRepo *git.Repository, path string, commit *git.Commit)
 		return GetDefaultTemplateConfig(), err
 	}
 
-	issueConfig := api.IssueConfig{}
+	issueConfig := GetDefaultTemplateConfig()
 	if err := yaml.Unmarshal(configContent, &issueConfig); err != nil {
 		return GetDefaultTemplateConfig(), err
```
Comment (delvh) [inline services/issue/template.go:77]: But if we initialize it to that, we can remove the second initialization here.

File: tests/integration/api_repo_file_helpers.go [hunk 2]
```diff
@@ -29,6 +30,32 @@ func createFileInBranch(user *user_model.User, repo *repo_model.Repository, tree
...
+}
+
+func createOrReplaceFileInBranch(user *user_model.User, repo *repo_model.Repository, treePath, branchName, content string) error {
+	_, err := deleteFileInBranch(user, repo, treePath, branchName)
+
+	if err != nil && !models.IsErrRepoFileDoesNotExist(err) {
+		return err
+	}
+
+	_, err = createFileInBranch(user, repo, treePath, branchName, content)
```
Comment (delvh) [inline tests/integration/api_repo_file_helpers.go:55]: Doesn't look like the fastest approach, but certainly a readable one.

File: services/issue/template.go (hunk 1 above)
Comment (JakobDev) [reply services/issue/template.go:77]: Does `yaml.Unmarshall` change something when it fails?

File: tests/integration/api_repo_file_helpers.go (hunk 2 above)
Comment (JakobDev) [reply tests/integration/api_repo_file_helpers.go:55]: I don't have much experience with the tests codebase. I just need a function that sets the content of a file. If you know a better way, you're welcome.

**Review by delvh — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**